### PR TITLE
migrating endpoints

### PR DESCRIPTION
**Pull request by stivni**

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -3544,6 +3544,61 @@ Delete a custom field definition
...
 + Response 204
 
+## Migrating [/migrate]
+
+WARNING! These endpoints are not guaranteed to stay backwards compatible.
+Their only use is to allow older integrations to migrate their data to the new API.
+
+### migrate.id [GET /migrate.id]
+
+This endpoint
```
Comment (jenssegers) [inline apiary.apib]: Text cut off?

**Comment by mrnnn** [comment]
What happened here? Commits appear double.

Also, we're only doing one-way? I feel we're gonna need both?

If they consume the new api and get a uuid, how are they gonna match it with their local data?

File: apiary.apib (hunk 1 above)
Comment (stivni) [reply apiary.apib]: WIP :) was going to explain why, but forgot while doing something else ;)

**Comment by stivni** [comment]
>  I feel we're gonna need both?
I would not allow that, because then there is no incentive to move to v2

> But what about the case where they consume the new api and get a uuid and need to match it with their local data? Or do we want them to retroactively replace all the id's on their side first before using the new api?

basically, yes, you need to fetch all new ids first if you want to start using api v2 with old data on your side... do you thing it's that weird?

**Pull request merged**